server/controller/userController.js

```diff
@@ -4,7 +4,26 @@ import requests from '../seedData/request';
  * @class Users
  */
 class Users {
-
+  /**
+ * @return {Object} all user requests
+ * @param {param} req
+ * @param {param} res
+ */
+  static retrieveRequests(req, res) {
+    if(requests.length === 0) {
+      return res.status(200).json({
+        status: 'fail',
```

Comment:
The status of 200 says it was successful which doesn't match with `status: 'fail'`

Reply:
According to my LFA in my previous bootcamp, he said that the status code should be a 200 for this kind of requests, since the request completed successfully, just that nothing was returned in the response object (In this case,when the requests array is empty).

Reply:
So it is not a failed request. . . It is a successful request with an empty array.